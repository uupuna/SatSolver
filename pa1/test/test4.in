1*0;
1*0
1*0; a:1,
1*0;a:1
(0+1)0;
(0+1)a;a:1
(0+1));