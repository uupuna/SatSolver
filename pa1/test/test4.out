1
Error: invalid output
1
1
